lib/puppet/util/character_encoding.rb

```diff
@@ -0,0 +1,96 @@
+# A module to centralize heuristics/practices for managing character encoding in Puppet
+require 'puppet/util'
```

Comment:
I suppose not. I think it was only in there as an aside during dev I think. The new `puppet/etc` does `require 'puppet/util/character_encoding'`, and its associated spec fails unless `puppet/util` is required first, but I can instead have `puppet/etc` require `puppet/util` instead of `puppet/util/character_encoding` which would effect the same thing and probably be less circuitous.